portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormRules.testcase

```diff
@@ -97,14 +97,61 @@ definition {
 	}
 
 	@description = "Verify that a Rule is autosaved when exiting the Form without saving the Rule"
-	@ignore = "True"
 	@priority = "5"
 	test AreSavedWhenExitingTheForm {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-134377 AreSavedWhenExitingTheForm pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Text");
+		Form.save();
+
+		FormsAdminNavigator.gotoRules();
+
+		FormRules.addRule();
+
+		FormRules.configureConditionOptions(
+			conditionField = "Text",
+			conditionOperator = "Is empty",
+			conditionRowNumber = "1");
+
+		FormRules.configureActionOptions(
+			action = "Require",
+			actionRowNumber = "1",
+			actionTarget = "Text");
+
+		FormsAdminNavigator.gotoBack();
+
+		FormsAdminNavigator.gotoEditFormByName(formName = "Form 1");
+
+		FormsAdminNavigator.gotoRules();
+
+		Click(locator1 = "FormRules#RULE_BUILDER_ADD_RULE_BUTTON");
+
+		FormRules.viewConfigureRules(
```

Comment:
@Diogomes, can we create two separated macros instead? Ideally, one will check the condition and the other one will check the action.